src/main/kotlin/RoutesParsing.kt

```diff
@@ -41,12 +46,15 @@ private fun TopicRouting(routeConfig: Toml): TopicRouting {
         else -> throw IllegalArgumentException("Invalid value for config parameter 'format'")
     }
     val verify: Boolean = routeConfig.getBoolean("verify", false)
-    val methods: Set<HttpMethod> = parseMethods(routeConfig)
-
-    return when(methods.isEmpty()) {
-        true -> TopicRouting(path, topic, format, verify)
-        false -> TopicRouting(path, topic, format, verify, methods)
+    val methods: Set<HttpMethod> = parseMethods(routeConfig).let {
+        if (it.isEmpty()) {
+            httpMethods.verbs
+        } else {
+            it
+        }
     }
+    val cors = parseCors(routeConfig)
+    return TopicRouting(path, topic, format, verify, methods, cors)
 }
 
 private fun parseMethods(toml: Toml): Set<HttpMethod> {
```

Comment:
I just realized we could just replace `emptyList()` with `httpMethods.verbs` and we can skip the let/if/else on line 49 to 54. Right?